Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -1716,4 +1716,21 @@ - (void)testPermissionOfferResponseTokenNotExist {
     [token stop];
 }
 
+#pragma mark - Validation
+
+- (void)testCompactOnLaunchCannotBeSet {
+    // This test just needs a single process, so skip child.
+    if (!self.isParent) { return; }
```

Comment:
The child process is created by `RLMRunChildAndWait()`, so there's no need to do anything other than just not call that if only one process is needed.